stat/distmv/statdist.go

```diff
@@ -335,15 +335,15 @@ func (Wasserstein) DistNormal(l, r *Normal) float64 {
 
 	// Compute Σ_l^(1/2)
 	var ssl mat.SymDense
-	ssl.PowPSD(&l.sigma, 0.5)
+	_ = ssl.PowPSD(&l.sigma, 0.5)
```

Comment:
Are we comfortable with this and the `PowPSD` call below? Do they deserved comments?

Reply:
Maybe same comment as above?

Reply:
Done